test/java/seedu/addressbook/data/AddressBookTest.java

```diff
@@ -135,4 +133,39 @@ public void getAllPersons() throws Exception {
 
         assertTrue(isIdentical(allPersons, personsToCheck));
     }
+
+    @Test
+    public void getSortedList() throws Exception {
+        UniquePersonList testPersonList  = new UniquePersonList(bobChaplin, aliceBetsy);
+        List<Person> unsortedList = testPersonList.listView();
+        List<Person> sortedList = testPersonList.sortedListView();
+        assertListNotSorted(unsortedList);
+        assertListSorted(sortedList);
+    }
+
+    public void assertListSorted(List<Person> inputList) {
+        List<Person> sortedList = sortList(inputList);
+        assertTrue(isIdentical(inputList, sortedList));
+    }
+
+    public void assertListNotSorted(List<Person> inputList) {
+        List<Person> sortedList = sortList(inputList);
+        assertFalse(isIdentical(inputList, sortedList));
+    }
```

Comment:
Is this an effective way to check if the list is really sorted? What if the list is randomly sorted, such that it is just different from the original input list? :P